stdlib/_typeshed/__init__.pyi

```diff
@@ -58,11 +58,11 @@ class SupportsItemAccess(SupportsGetItem[_KT_contra, _VT], Protocol[_KT_contra,
     def __setitem__(self, __k: _KT_contra, __v: _VT) -> None: ...
     def __delitem__(self, __v: _KT_contra) -> None: ...
 
-# StrPath and AnyPath can be used in places where a
-# path can be used instead of a string, starting with Python 3.6.
+# These aliases are simple strings in Python 2.
 StrPath = Union[str, PathLike[str]]
 BytesPath = Union[bytes, PathLike[bytes]]
-AnyPath = Union[str, bytes, PathLike[str], PathLike[bytes]]
+StrOrBytesPath = Union[str, bytes, PathLike[str], PathLike[bytes]]
+AnyPath = StrOrBytesPath  # obsolete, will be removed soon
```

Comment:
I thought you were planning to turn this into an AnyStr-like TypeVar

Reply:
I was, but I wanted to give this a bit of deprecation, then a bit of removal, then a resurrection (with about one mypy release between each), just to give people a change to react.

Reply:
Also the next mypy release will cause lots of problems, and I want to avoid to add this one to it.